releases/release-1.22/release-notes/maps/pr-102489-map.yaml

```diff
@@ -0,0 +1,3 @@
+pr: 102489
+releasenote:
+  text: Fixed 'kubelet' runtime panic for timed-out portforward streams.
```

Comment:
I would like to see if this needs to be removed from the Release notes, because in https://github.com/kubernetes/kubernetes/pull/102838 it was reverted. I think it can stay but we should talk about it at the meeting. What do the team think about this:
@Damans227 @simcard0000 @sladyn98 
Feel free to corrected if this was not reverted ofc :)

Reply:
Thanks for sharing the info. How would we generally handle those revert event? Should we keep it here and also keep the future  kubernetes/kubernetes#102838 release note or we delete both? I believe keeping it both might make more sense.

Reply:
I see value in both, I think we can add something at the end like:

(reverted in #102838)